README.md

```diff
@@ -64,3 +64,105 @@ $ docker run --volume $PWD:/home/harambe/project --volume ~/.tokenlists:/home/ha
 This project is in development and should be considered a beta.
 Things might not be in their final state and breaking changes may occur.
 Comments, questions, criticisms and pull requests are welcomed.
+
+## Full Environment Execution Example
+
+Running the `Quick Usage` and `Docker Usage` examples will fail if you do not have a full environment setup.
+
+First, it is suggested that you use a virtual environment, and then install the Silverback application. Choose any of your liking. This is not a requirement, but will help with setup if you do not have a `~/.tokenslists` folder.
```

Comment:
This suggestion on using a venv should appear underneath the "quick start" section (right under the note)

You can keep the part about `~/.tokenlists/`